development/base/bot.py

```diff
@@ -0,0 +1,68 @@
+import base.settings as sett
+
+class Bot:
```

Comment:
Could you add the unit testing?

https://habr.com/ru/company/otus/blog/481806/
https://pythonworld.ru/moduli/modul-unittest.html